data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/file/FileWatcher.java

```diff
@@ -107,7 +119,8 @@ public synchronized void close() throws Exception {
     private void run() {
         try {
             // register the given watch service.
-            // Note: this watch a directory and not the single file we're interested in, so that's the
+            // Note: this watch a directory and not the single file we're interested in, so
+            // that's the
             // reason in #watch() we filter watch service events based on the updated file.
```

Comment:
It's a little weird the broken line of the comment 
"the <ton of space>
reason"